lib/mining/miner.js

```diff
@@ -11,6 +11,7 @@ const assert = require('bsert');
 const EventEmitter = require('events');
 const Heap = require('bheep');
 const {BufferMap} = require('buffer-map');
+const rng = require('bcrypto/lib/random');
```

Comment:
Any reason this is called `rng` here but `random` elsewhere?

Reply:
It was possibly due to the length of `random` caused linting issues whereas `rng` is shorter and doesn't. Updated to use `random` and there isn't a problem.